### ISSUE TITLE
Sync failed with error: "expected date for field "last_sync_at" but got "٢٠٢٠-١٠-٢٧T١٤:١٤:١٨.٩٣٠Z""

### ISSUE THREAD

**Issue by harrunisk** [GetStream/stream-chat-android#760]
**Describe the bug**
When we change the app language to Arabic we are getting this error frequently.  We haven't updated SDK to the latest version yet. But I didn't see any changelog about this crash.
```
Fatal Exception: java.lang.IllegalStateException: Status code: 400, with stream code: 4, description: Sync failed with error: "expected date for field "last_sync_at" but got "٢٠٢٠-١٠-٢٧T١٤:١٤:١٨.٩٣٠Z""
       at io.getstream.chat.android.livedata.ChatDomainImpl.queryEvents(ChatDomainImpl.kt:495)
       at io.getstream.chat.android.livedata.ChatDomainImpl.replayEventsForActiveChannels(ChatDomainImpl.kt:519)
       at io.getstream.chat.android.livedata.ChatDomainImpl.replayEventsForActiveChannels$default(ChatDomainImpl.kt:506)
       at io.getstream.chat.android.livedata.ChatDomainImpl.connectionRecovered(ChatDomainImpl.kt:569)
       at io.getstream.chat.android.livedata.ChatDomainImpl$connectionRecovered$1.invokeSuspend(:12)
       at kotlin.coroutines.jvm.internal.BaseContinuationImpl.resumeWith(ContinuationImpl.kt:33)
       at kotlinx.coroutines.DispatchedTask.run(DispatchedTask.kt:56)
       at kotlinx.coroutines.scheduling.CoroutineScheduler.runSafely(CoroutineScheduler.kt:571)
       at kotlinx.coroutines.scheduling.CoroutineScheduler$Worker.executeTask(CoroutineScheduler.kt:738)
       at kotlinx.coroutines.scheduling.CoroutineScheduler$Worker.runWorker(CoroutineScheduler.kt:678)
       at kotlinx.coroutines.scheduling.CoroutineScheduler$Worker.run(CoroutineScheduler.kt:665)
```

**SDK version**
- 4.3.1-beta-1

**To Reproduce**
1. Change the app language to Arabic
2. Set user
3. Hang out in chat

**Device:**
 - Redmi Note 5 Pro-Android 9
 - General Mobile GM 8-Android 7.1.1

**Comment by zsmb13** [comment]
We have a fix for an issue like this in version 4.4.0, could you please try that latest version of the SDK?

**Comment by JcMinarro** [comment]
Hello @harrunisk 
It was fixed on #754  and it is already integrate into our [last release](https://github.com/GetStream/stream-chat-android/releases/latest)
I'm closing this issue now

**Comment by harrunisk** [comment]
We updated to the 4.4.0 version. Everything is fine. Thanks for your efforts.
